tutorial-content/content/weighting-schemes.md

```diff
@@ -255,6 +273,19 @@ Start by throwing away extraneous data, by cropping the LandScan to, say,
 landscan <- crop(landscan, extent(-126, -66, 24, 50))
 ```
 ````
+````{tab-item} Python
+```Python
+from rasterio.windows import from_bounds
+
+xmin, xmax, ymin, ymax = -126, -66, 24, 50
+
+# Calculate the window to crop
+window = from_bounds(xmin, ymin, xmax, ymax, src.transform)
+
+# Read the data from this window
+landscan = src.read(1, window=window)
```

Comment:
You should have some text here since this is organized differently from R. In R, we opened the raster in the previous code block. For python, you want to say something like, "In Python, you can impose the window when you read the raster file. We will open the raster file again here, for completeness." Then you should do the with... line again.

Reply:
Actually, I see that landscan can also do this `.read(..., window)` thing. So just use landscan rather than src.